joshuaCarpentier/assignments/treesDFS/LC257/LC257.js

```diff
@@ -0,0 +1,65 @@
+// Problem Statement #
+
+// Given the root of a binary tree, return all root-to-leaf paths in any order.
+// A leaf is a node with no children.
+
```

Comment:
Make sure you are including an overview of your approach here. Nice job attempting to write additional tests but it seems like they are incomplete. Also make sure to analyze the time and space complexity of each problem.